packages/cyberstorm/src/components/PaginationButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import styles from "./componentStyles/PaginationButton.module.css";
+
+export interface PaginationButtonProps {
+  isSelected?: boolean;
+  ariaCurrent?: boolean;
+  ariaLabel?: string;
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  colorScheme?: "default";
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+export const PaginationButton: React.FC<PaginationButtonProps> = (props) => {
+  const {
+    ariaCurrent,
+    ariaLabel,
+    label,
+    leftIcon,
+    rightIcon,
+    colorScheme,
+    onClick,
+    isSelected,
+  } = props;
+  return (
+    <button
+      type="button"
+      aria-current={ariaCurrent ? "page" : false}
```

Comment:
Only the current page needs the aria attribute attached - the default is `false` so there's no need to explicitly state that each time.